### PR TITLE
issue #1374: myNotes refactoring

### PR DESCRIPTION
**Pull request by kojoty**
This PR contains refactoring around cache notes - especially cache_notes list and more.
The view is a little raw so I will try to polish it on in next days. 
Additionally final change will need change at OKAPI side - which I will need to prepare.

@deg-pl, @andrixnet, @harrieklomp 
This PR contains sqlalter which should be applied after merge (I will call you again).

@harrieklomp, @andrixnet 
this PR contains also translation changes.

I'm going to merge it soon (I think tomorrow evening).

### CODE REVIEW

File: Utils/Uri/SimpleRouter.php [hunk 1]
```diff
@@ -45,12 +45,16 @@ public static function getLink($ctrl, $action=null, $params=null)
...
-            $action = 'index';
-        }
+        $link = "/$ctrl";
 
-        //$link = '/index.php?'.self::ROUTE_GET_VAR."=/$ctrl/$action";
-        $link = "/$ctrl/$action";
+        if($action){
+            $link .= "/$action";
+        }else{
+            if($params){
```
Comment (deg-pl) [inline Utils/Uri/SimpleRouter.php]: Ten warunek jest za słaby. Jeśli $params = 0, to całość zostanie uznana za fałsz i parametr nie zostanie dodany. Sprawdzone na moim nowym kodzie :)

Comment (deg-pl) [reply Utils/Uri/SimpleRouter.php]: Ech - tu jest $action, a nie $params, ale sytuacja jest analogiczna tu i tu.

Comment (kojoty) [reply Utils/Uri/SimpleRouter.php]: should be fixed by 1259d4c

**Comment by andrixnet** [comment]
Will translate after sqlalter.

**Comment by kojoty** [comment]
@deg-pl, @andrixnet, @harrieklomp,
please remember about sqlalter from this PR

**Comment by deg-pl** [comment]
Done on OC PL and OC UK.
Please remember to update opencaching-pl/docs/database/ also.

**Comment by andrixnet** [comment]
sqlAlter done at OCRO

**Comment by harrieklomp** [comment]
Done on OCNL

**Pull request merged**